### ISSUE TITLE
runtime: Decide on whether to also support zlib in addition to zstandard

### ISSUE THREAD

**Issue by probonopd** [probonopd/static-tools#34]
Should the runtime also support zlib in addition to zstandard?

Only containing zstandard:

![image](https://user-images.githubusercontent.com/2480569/190851408-d47c8776-51bb-48cb-af17-9571f0ec4640.png)

Also containing zlib:

(tnd)

**Comment by xplshn** [comment]
Why not ZSTD?

**Comment by probonopd** [comment]
Yes, zstd is preferred. The question is whether we should allow another option as well.
